README.md

```diff
@@ -2,16 +2,6 @@
 description: 'The native NVMe-oF CAS engine of OpenEBS'
 ---
 
-{% hint style="danger" %}
-### This documentation is staged on an unstable branch  
-
-It is provided here as a convenience to the writers, reviewers and editors of Mayastor's user documentation, to provide easy visualisation of content before publishing.  It MUST NOT be used to guide the installation or use of Mayastor, other than for pre-release testing outside of production.  As a staging branch, it is expected at times to contain errors and to be incomplete.  
-
-The official Mayastor User Reference for the latest GA release can be found [here](https://mayastor.gitbook.io/) 
-{% endhint %}
-
-# Welcome to Mayastor!
-
```

Comment:
This warning text is supposed to exist for "develop" as "develop" can be unstable. It needs to be removed for all release versions only.